scripts/delete-dev.sh

```diff
@@ -109,6 +109,16 @@ while [ $# -gt 0 ]; do
   shift 1
 done
 
+# Confirm before deleting all resources
+echo ""
+echo -n "Are you sure you want to delete all development resources? [y/N]: "
+read -r confirm
+confirm=$(echo "$confirm" | tr '[:upper:]' '[:lower:]')
+if [ "$confirm" != "y" ]; then
```

Comment:
Instead of always running interactively, consider adding a `-y` or `--yes` flag to allow non-interactive execution for automation purposes.